modules/core/src/main/scala/org/scalasteward/core/edit/UpdateHeuristic.scala

```diff
@@ -171,7 +171,7 @@ object UpdateHeuristic {
   val sliding = UpdateHeuristic(
     name = "sliding",
     replaceVersion = defaultReplaceVersion(
-      _.mainArtifactId.toSeq.sliding(5).map(_.unwrap).take(5).filterNot(_ === "scala").toList
+      _.mainArtifactId.toSeq.sliding(5).map(_.unwrap).take(20).filterNot(_ === "scala").toList
```

Comment:
That's interesting! I'm wondering why `take(5)` was there in the first place? Maybe we can get rid of the `take` call completely.

Reply:
that's right ! I've removed it. Probably someone got confused with sliding(5).

Reply:
It was there ever since this heuristic is in Scala Steward: https://github.com/scala-steward-org/scala-steward/pull/527. Unfortunately I didn't mention why I've added it. I guess I thought that only words at the beginning of artifacIds should be used as search terms for the version. But I'm OK with removing this restriction now.